packages/website/src/hooks/cannon.ts

```diff
@@ -385,20 +388,40 @@ export function useCannonPackage(packageRef: string, chainId?: number) {
     enabled: !!pkgUrl,
   });
 
+  const fullPackageRef = ipfsQuery.data?.fullPackageRef;
+
+  const registryQueryMeta = useQuery({
+    queryKey: ['cannon', 'registry-meta', fullPackageRef, packageChainId],
+    queryFn: async () => {
+      if (typeof fullPackageRef !== 'string' || fullPackageRef.length < 3) {
+        return null;
+      }
+
+      const metaUrl = await registry.getMetaUrl(fullPackageRef, packageChainId);
+
+      if (metaUrl) {
+        return { metaUrl };
+      } else {
+        throw new Error(`package not found: ${fullPackageRef} (${packageChainId})`);
+      }
+    },
+    refetchOnWindowFocus: false,
```

Comment:
does it make sense to disable it globally when Tanstak/Query is initialized.

I've seen this refech in many places and it is quite odd. I think the default should be disabled and enabled when required on specific quieres.

Reply:
Yup, I think you are right, best would be to disable it globally. We only really need it on the signatures list, when signing a transaction.